src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -2,12 +2,33 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
+    <AvailablePlatforms>amd64,x86,arm,armel,arm64</AvailablePlatforms>
     <ProjectGuid>{B1BF7CE0-CAB5-4FA2-A39C-450B05D5DB1C}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>1718</NoWarn>
     <DefineConstants Condition="'$(TargetGroup)'=='netcoreapp'">$(DefineConstants);netcoreapp</DefineConstants>
     <DefineConstants Condition="'$(TargetGroup)'=='uapaot'">$(DefineConstants);uapaot</DefineConstants>
   </PropertyGroup>
+  <!-- Platform specific properties -->
+  <PropertyGroup Condition="'$(Platform)' == 'amd64'">
```

Comment:
Do we build tests as platform specific anywhere else in CoreFX?

If we are not doing it anywhere else, this is likely going to introduce a whole bunch of problems that will be pain to deal with.

Note that you can check for bitness at runtime. It is what we do pretty much everywhere else in CoreFX.

Reply:
thanks @jkotas yes, I wasnt sure. I grabbed this from coreclr to test it out on the build.